.envrc

```diff
@@ -1 +1,11 @@
-use flake
+{
+# shell gc root dir
+mkdir -p $(direnv_layout_dir)
+
+# reload when these files change
+watch_file flake.nix
+watch_file flake.lock
+
+# load the flake devShell
+eval "$(nix print-dev-env --impure --profile $(direnv_layout_dir)/flake-profile)"
+} || use nix
```

Comment:
Can we somehow explicitly declare this dependency or does it make sense to at least document it in the README? I'm not sure if we want to maintain this on our own if there's an existing tool for that :)

Reply:
direnv does have a mechanism for for import nix-direnv if it doesn't exist. I just figured, since the functionality can be replicated in a few lines that it wasn't worth it.